resources/carrier/fa/98.txt

```diff
@@ -21,10 +21,18 @@
 98931|اسپادان
 98932|تالیا
 98933|ایرانسل
-98934|کیش
+98934|تله‌کیش
 98935|ایرانسل
 98936|ایرانسل
 98937|ایرانسل
 98938|ایرانسل
 98939|ایرانسل
-98990|همراه اول
\ No newline at end of file
+98990|همراه اول
+98991|همراه اول
+9899810|شاتل موبایل
+989944|انارستان
```

Comment:
Please move this one line above so that it is in right order (i.e above "9899810" entry)